src/main/js/services/provider/AuthProvider.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import {authMethods} from "../firebase/authMethods";
+
+export const firebaseAuth = React.createContext();
+
+const AuthProvider = (props) => {
+    const initState = {email: '', password: ''}
+    const [inputs, setInputs] = useState(initState)
+    const [errors, setErrors] = useState([])
+    const [token, setToken] = useState(null)
+
+    const handleSignUp = () => {
+        authMethods.signUp(inputs.email, inputs.password,setErrors ,setToken )
+        console.log(errors, token)
```

Comment:
Do you think it would be better to use react-notification for any errors handling in Sign-Up/Sign-in

Reply:
It could be a great way to notify users. I'll review that part and update my pull request. Thanks a lot!! 😄